modules/mobile/app/models/mobile/v0/service_graph.rb

```diff
@@ -31,9 +31,10 @@ def affected_services(windows)
 
           while queue.size != 0
```

Comment:
Am I crazy, or is this while loop just an each loop in disguise?

Reply:
haha yeah, even tho it's ugly I didn't wanna touch it in fear of introducing some differing logic since that's not technically what's wrong with the code.